/fslhome/tarch/.conda/envs/tf16/bin/python
2020-03-07 20:52:25.144224: E tensorflow/stream_executor/cuda/cuda_driver.cc:406] failed call to cuInit: CUDA_ERROR_NO_DEVICE
